portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+</head>
```

Comment:
Indent here so that the formatting is correct

Reply:
Fixed (shown in later commit)